backend/gateway/mux/mux.go

```diff
@@ -180,6 +180,20 @@ func customResponseForwarder(ctx context.Context, w http.ResponseWriter, resp pr
 	return nil
 }
 
+// https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/customizing_your_gateway/#mapping-from-http-request-headers-to-grpc-client-metadata
+// adds custom rules for the Slack API headers that would otherwise be dropped as they don't meet the preconditions of the default rules
+func customMatcher(key string) (string, bool) {
+	switch key {
+	case "X-Slack-Signature":
+		return key, true
+	case "X-Slack-Request-Timestamp":
+		return key, true
```

Comment:
i would forward all X- headers with the grpcgateway- prefix, no need to make this specific to slack really. in our case we could forward all headers really, but starting with x- is fine for now.

let's also put it in a separate PR.

Reply:
ok, will move out of this PR